apps/api/drizzle/meta/_journal.json

```diff
@@ -108,4 +108,4 @@
       "breakpoints": true
     }
   ]
-}
```

Comment:
The files in the drizzle directory did not pass formatting.  I updated them with `pnpm format`, but they don't have anything to do with this PR.